airflow/bin/cli.py

```diff
@@ -469,7 +473,10 @@ def run(args, dag=None):
             DeprecationWarning)
         remote_base = conf.get('core', 'S3_LOG_FOLDER')
 
-    if os.path.exists(filename):
+    if remote_base == 'None':
```

Comment:
The if just below this line will check empty strings. So that's covered as well.